Sources/SmileID/Classes/Metadata/DeviceOrientationMetadataProvider.swift

```diff
@@ -53,7 +57,9 @@ class DeviceOrientationMetadataProvider: MetadataProvider {
     }
 
     func addDeviceOrientation() {
-        deviceOrientations.append(currentOrientation)
+        deviceOrientations.append(
+            OrientationEvent(value: currentOrientation, date: Date())
```

Comment:
```suggestion
            OrientationEvent(value: currentOrientation)
```
If we do the above, we can simplify this line to just set the orientation